src/scan/postgres_seq_scan.cpp

```diff
@@ -98,7 +100,7 @@ PostgresSeqScanFunction::PostgresSeqScanFunc(duckdb::ClientContext &context, duc
 
 	auto hasTuple = local_state.m_heap_table_reader->ReadPageTuples(output);
 
-	if (!hasTuple || local_state.m_heap_table_reader->GetCurrentBlockNumber() == InvalidBlockNumber) {
+	if (!hasTuple || IsValidBlockNumber(local_state.m_heap_table_reader->GetCurrentBlockNumber())) {
```

Comment:
Probably the reason for the failing tests:
```suggestion
	if (!hasTuple || !IsValidBlockNumber(local_state.m_heap_table_reader->GetCurrentBlockNumber())) {
```

Reply:
Ah yeah that would help :-) - I'm waiting for #421 to land to take care of this one